pywinauto/base_application.py

```diff
@@ -407,149 +622,48 @@ def exists(self, timeout=None, retry_interval=None):
             criterion['visible'] = None
 
         try:
-            self.__resolve_control(exists_criteria, timeout, retry_interval)
+            self.find(timeout, retry_interval)
 
             return True
         except (findwindows.ElementNotFoundError,
                 findbestmatch.MatchError,
                 controls.InvalidWindowHandle,
-                controls.InvalidElement):
+                controls.InvalidElement,
+                TimeoutError):
             return False
 
-    @classmethod
-    def __parse_wait_args(cls, wait_conditions, timeout, retry_interval):
-        """Both methods wait & wait_not have the same args handling"""
+    def not_exists(self, timeout=None, retry_interval=None):
+        """
+        Check if the window exists, return True if the control does not exists`
+
+        :param timeout: the maximum amount of time to wait for the
+                    control to exists. Defaults to ``Timings.exists_timeout``
+        :param retry_interval: The control is checked for existance this number
```

Comment:
It would be good to take these params description from old doc strings as they have proven good enough English. ;)